spec/controllers/account/page_controller_spec.rb

```diff
@@ -3,14 +3,15 @@
 RSpec.describe Account::PagesController, type: :controller do
   context 'Page' do
     let!(:page) { FactoryBot.create(:page) }
+    before(:each) { sign_in page.company.user }
 
-    it 'Get#index' do
-      get :index
+    it 'GET #index' do
+      get :index, params: { company_id: page.company }
```

Comment:
incorrect param key `:company_id`
you should pass `company_id: page.company.to_param` instead of `ActiveRecord` instance